monkey/common/types.py

```diff
@@ -14,6 +17,12 @@
 MachineID: TypeAlias = PositiveInt
 
 
+@dataclass
+class PingScanData:
+    response_received: bool
+    os: Optional[OperatingSystem]
+
+
```

Comment:
IMO this whole common folder should be re-organized. It contains pydantic models (data structures) and common logic. These things should be separated. `PingScanData` should ideally be in `/common/models/fingerprinting/ping_scan_data.py` or something. We'll be adding a lot more of these models and we're making a mess

Reply:
Maybe we make an issue about reorganizing ?